src/pages/Admin/Venue/VenueAdminPage.tsx

```diff
@@ -58,3 +59,12 @@ export const VenueAdminPage: React.FC = () => {
     </>
   );
 };
+
+export const VenueAdminPage = () => {
+  const venueId = useVenueId();
+  return (
+    <RelatedVenuesProvider venueId={venueId}>
```

Comment:
**Note**: It may look like OOS for the title of this PR, but it does help fix access to the `VenueAdminPage` in a sense there isn't BSOD and/or error.

Reply:
I changed the PR title to be more generic anyway:

> fix Error: `<WorldUsersProvider/>` not found in a more holistic way